lib/unit-ruby/util/connection.rb

```diff
@@ -47,6 +47,17 @@ def post(path, data = nil)
       from_json_api(response.body)
     end
 
+    # Executes a DELETE request to the API
+    #
+    # @return boolean
+    def delete(path)
+      response = connection.delete(path)
+
+      handle_errors(response)
+
+      true
```

Comment:
Does the API return json like the other types of requests?

```suggestion
      from_json_api(response.body)
```

Reply:
Yeah it returns the resource again but with a status of `Deleted`- I'm updating it to return from_json_api 👍